frontend/src/utils/Helpers/getAuthResponseData.tsx

```diff
@@ -0,0 +1,6 @@
+import Cookies from 'js-cookie';
+
+export const getAuthResponseData = (cookieName: string): string | undefined => {
+  const backendData: string | undefined = Cookies.get(cookieName);
+  if (backendData) return JSON.parse(backendData);
```

Comment:
Why helpers from uppercase?
Keep please clear structure of `if` statement!

Reply:
fixed